src/plugins/dashboard/public/application/top_nav/dashboard_top_nav.tsx

```diff
@@ -597,6 +597,8 @@ export function DashboardTopNav({
                   data-test-subj="dashboardAddNewPanelButton"
                 />
               ),
+              controlsButton:
+                dashboardAppState.dashboardContainer.controlGroup?.getToolbarButtons(),
```

Comment:
Note that, if the new controls are not enabled in Labs, then `controlGroup` won't be defined - therefore, this toolbar button *won't* be rendered. No extra checks necessary - cool! 👍 (This is more of a note to myself since I was confused at first, LOL 😄)